assets/app/features/upgrade/i18n/en.json

```diff
@@ -61,19 +61,19 @@
             },
             "database-configuration": {
                 "name": "",
-                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Database for the OpenStack Database Migration.",
+                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Server for the OpenStack Database Migration.",
                 "navigation-tabs": {
-                    "create-database": "Create New Database",
-                    "connect-database": "Connect to an Existing Database"
+                    "create-server": "Create New Server",
```

Comment:
We need to align to Ken's copys as specified in: https://github.com/crowbar/crowbar-ui/wiki/Upgrade67-Create-or-Connect-To-PostgreSQL-Database